test/framework/src/main/java/org/elasticsearch/search/aggregations/AggregatorTestCase.java

```diff
@@ -741,15 +737,25 @@ private void writeTestDoc(MappedFieldType fieldType, String fieldName, RandomInd
             if (typeName.equals(BinaryFieldMapper.CONTENT_TYPE)) {
                 doc.add(new BinaryFieldMapper.CustomBinaryDocValuesField(fieldName, new BytesRef("a").bytes));
                 json = "{ \"" + fieldName + "\" : \"a\" }";
-            } else if (typeName.equals(IpFieldMapper.CONTENT_TYPE)) {
-                // TODO note: once VS refactor adds IP, this conditional will go away
-                InetAddress ip = randomIp(randomBoolean());
-                json = "{ \"" + fieldName + "\" : \"" + NetworkAddress.format(ip) + "\" }";
-                doc.add(new SortedSetDocValuesField(fieldName, new BytesRef(InetAddressPoint.encode(ip))));
             } else {
                 doc.add(new SortedSetDocValuesField(fieldName, new BytesRef("a")));
                 json = "{ \"" + fieldName + "\" : \"a\" }";
             }
+        } else if (vst.equals(CoreValuesSourceType.DATE)) {
```

Comment:
I believe we lost the check for date_nanos (`DateFieldMapper.DATE_NANOS_CONTENT_TYPE)`) which I think will eventually break?

We can do a quick follow-on PR instead of changing this one imo.